frontend/src/Components/Sidebar.jsx

```diff
@@ -79,6 +80,10 @@ const Sidebar = () => {
 				<RiDashboardFill />
 				<div>Dashboard</div>
 			</PageLink>
+			<PageLink to="/lamps" end>
```

Comment:
```suggestion
			<PageLink to="/lamps">
```
This wouldn't work if we had more pages under /lamps